cmd/common.go

```diff
@@ -336,6 +336,20 @@ func isThisNodeRunning(nodeType check.NodeType) bool {
 	return true
 }
 
+func exitCodeSelection(controlsCollection []*check.Controls) int {
+	checksFailed := false
+	for _, control := range controlsCollection {
+		if control.Fail > 0 {
+			checksFailed = true
+		}
+	}
+
+	if checksFailed && exitCode != 0 {
+		return exitCode
+	}
+	return 0
+}
```

Comment:
You might simplify this like so:
```suggestion
func exitCodeSelection(controlsCollection []*check.Controls) int {
	for _, control := range controlsCollection {
		if control.Fail > 0 {
			return exitCode
		}
	}

	return 0
}
```

Reply:
Yup totally agree, I've changed it to that now